MSOLSpray.py

```diff
@@ -34,7 +34,7 @@
 force = args.force
 out = args.out
 verbose = args.verbose
-sleep = args.sleep
+sleep = int(args.sleep)
```

Comment:
The typecast to `int` isn't needed, because argparse casts it when using `type=int`.